regression/tests/enterprise/test_ent_verify_sso_id.py

```diff
@@ -0,0 +1,84 @@
+"""
+Enterprise SSO ID Verification tests
+"""
+from regression.tests.enterprise.ent_test_base import EnterpriseTestBase
+from regression.pages.whitelabel.basket_page import (
+    SingleSeatBasketPage
+)
+from regression.pages.enterprise.enterprise_const import (
+    ENT_COURSE_ID
+)
+
+
+class TestEnterpriseSSOIDVerification(EnterpriseTestBase):
+    """
+    Test Enterprise ID Verification page
+    """
+
+    def setUp(self):
+        """
+        Initialize all page objects
+        """
+        super(TestEnterpriseSSOIDVerification, self).setUp()
+
+    def test_enterprise_user_id_verification_status(self):
+        """
+        Scenario: To verify that enterprise user ID automatically verified
+         during SSO process
+            Given that a user is on data consent page
+            When this user checks the data consent check box
+            And Clicks on the Continue button
+            Then this user is taken to basket page
+            complete payment process
+            Then user should not see any ID verification panel
+            on receipt page as enterprise learners ID
+            automatically verified if enterprise SSO settings enabled
+            in admin configuration.
+        """
+        self.ecommerce_courses_page.visit()
```

Comment:
why do we need to visit `ecommerce_courses_page` ?

Reply:
The eCommerce page is visited just to make sure that when user jumps to eCommerce from SAP portal we don't have to handle authentication popup.